packages/angular/src/generators/init-mfe/schema.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "GeneratorAngularMFEInit",
+  "cli": "nx",
+  "title": "Init MFE Setup for Angular App",
+  "description": "Create MFE config files to allow Angular App to use Module Federation",
+  "type": "object",
+  "properties": {
+    "appName": {
+      "type": "string",
+      "description": "The name of the app to generate the MFE App Config for",
```

I think the following might read better:

```suggestion
      "description": "The name of the application to generate the MFE configuration for.",
```

The main things, that would apply to other descriptions in this schema are:

- `application` instead of `app`.
- `configuration` instead of `config` and in lowercase.
- The end point of the sentences.

I think in general, the docs look more professional/polished when we use the full words. I might be wrong of course 🙂

Let me know your thoughts.